src/Pulsar.Client/Common/DTO.fs

```diff
@@ -117,17 +117,36 @@ type Broker =
         PhysicalAddress: PhysicalAddress
     }
 
+type CompressionType =
+    | None = 0
+    | LZ4 = 1
+    | ZLib = 2
+    | ZStd = 3
+    | Snappy = 4
 
 type Metadata =
     {
         NumMessages: int
         HasNumMessagesInBatch: bool
+        CompressionType: CompressionType
+        UncompressedMessageSize: int32
```

Comment:
Can you please add UncompressedMessageSize to UMX?

Reply:
It don't see the point.